layerstats/README.md

```diff
@@ -0,0 +1,175 @@
+| x                   | tile x                                                                                                                                          |
+| y                   | tile y                                                                                                                                          |
+| hilbert             | tile hilbert ID (defines [pmtiles](https://protomaps.com/docs/pmtiles) order)                                                                   |
+| archived_tile_bytes | stored tile size (usually gzipped)                                                                                                              |
+| layer               | layer name                                                                                                                                      |
+| layer_bytes         | encoded size of this layer on this tile                                                                                                         |
+| layer_features      | number of features in this layer                                                                                                                |
+| layer_attr_bytes    | encoded size of the [attribute key/value pairs](https://github.com/mapbox/vector-tile-spec/tree/master/2.1#44-feature-attributes) in this layer |
+| layer_attr_keys     | number of distinct attribute keys in this layer on this tile                                                                                    |
+| layer_attr_values   | number of distinct attribute values in this layer on this tile                                                                                  |
+
+### Analyzing Layer Stats
+
+Load a layer stats file in [duckdb](https://duckdb.org/):
+
+```sql
+create table layerstats as select * from 'output.pmtiles.layerstats.tsv.gz';
+```
+
+Then get the biggest layers:
+
+```sql
+select * from layerstats order by layer_bytes desc limit 2;
+```
+
+| z  |  x   |  y   |  hilbert  | archived_tile_bytes |  layer   | layer_bytes | layer_features | layer_attr_bytes | layer_attr_keys | layer_attr_values |
+|----|------|------|-----------|---------------------|----------|-------------|----------------|------------------|-----------------|-------------------|
+| 14 | 6435 | 8361 | 219723809 | 679498              | building | 799971      | 18             | 68               | 2               | 19                |
+| 14 | 6435 | 8364 | 219723850 | 603677              | building | 693563      | 18             | 75               | 3               | 19                |
+
+To get a table of biggest layers by zoom:
+
+```sql
+pivot (
+  select z, layer, (max(layer_bytes)/1000)::int size from layerstats group by z, layer order by z asc
+) on z using sum(size);
+```
+
+|        layer        |  0  |  1  | 10  | 11  | 12  | 13  | 14  |  2  |  3  |  4  |  5  | 6  |  7  |  8  |  9  |
```

Comment:
can we make this sorted numerically instead of lexicographically?

Reply:
```
PIVOT ... ON printf('%2d', z)
```

🙃